src/overlays/actors/ovl_En_Az/z_en_az.c

```diff
@@ -39,108 +83,1755 @@ const ActorInit En_Az_InitVars = {
+                    case 4329:
+                        func_80A98EFC(this, play, 0x10EA, 0xC, 4);
+                        ret = 0;
+                        break;
+                    case 4330:
+                        func_80A98EFC(this, play, 0x10EB, 0, 4);
+                        ret = 0;
+                        break;
+                    case 4331:
+                        if (play->msgCtx.choiceIndex == 0) {
+                            play->msgCtx.msgMode = 0x44;
+                            func_8019F208();
+                            switch (this->unk_2FA) {
+                                case 4:
+                                    this->unk_2FA = 3;
+                                    break;
+                                case 7:
+                                    this->unk_2FA = 6;
+                                    break;
+                                case 9:
+                                default:
+                                    this->unk_2FA = 8;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Az/func_80A97114.s")
+                                    break;
+                            }
+                            ret = 0;
+                        } else {
+                            func_8019F230();
+                            this->actor.textId = 0x10EC;
+                            SubS_ChangeAnimationBySpeedInfo(&this->skelAnime, D_80A99010, 7, &this->animIndex);
+                            SubS_ChangeAnimationBySpeedInfo(&brother->skelAnime, D_80A99010, 0xD, &brother->animIndex);
+                        }
+                        break;
+                    case 4332:
+                        func_80A98EFC(this, play, 0x10ED, -1, 6);
+                        ret = 0;
+                        break;
+                    case 4333:
+                        if ((this->unk_2FA == 4) || (this->unk_2FA == 9)) {
+                            gSaveContext.save.weekEventReg[24] &= (u8)~4;
+                        }
+                        func_80A94AB8(this, play, 0);
+                        func_80A979DC(this, play);
+                        ret = 0;
+                        break;
+                    case 4334:
+                        func_80A98EFC(this, play, 0x10EF, 0xC, 0xA);
+                        ret = 0;
+                        break;
+                    case 4335:
+                        func_80A98EFC(this, play, 0x10F0, 0xD, 9);
+                        ret = 0;
+                        break;
+                    case 4336:
+                        func_80A98EFC(this, play, 0x10F1, 0xC, 0xA);
+                        ret = 0;
+                        break;
+                    case 4337:
+                        gSaveContext.save.weekEventReg[93] |= 1;
+                        if (gSaveContext.save.weekEventReg[23] & 0x80) {
+                            this->unk_3CC = 4;
+                        } else {
+                            this->unk_3CC = 90;
```

Comment:
```suggestion
                            this->getItemId = GI_RUPEE_RED;
                        } else {
                            this->getItemId = GI_BOTTLE;
```
This entry is for `getItemId`'s, could you fill out the enums for this throughout?